parlai/core/dict.py

```diff
@@ -818,6 +850,26 @@ def __str__(self):
         """
         return str(self.freq)
 
+    def sync_bytelevelbpe_dict(self):
+        """
+        This will sync the dict agent and the hugging face bytelevel bpe dict.
+        """
+        if self.tokenizer != 'bytelevelbpe':
+            return
+        special_tokens = [
+            self.start_token,
+            self.end_token,
+            self.unk_token,
+            self.null_token,
+        ]
+        self.byte_level_bpe.tokenizer.add_special_tokens(special_tokens)
+        self.ind2tok = {}
+        self.tok2ind = {}
+        for i in range(self.byte_level_bpe.tokenizer.get_vocab_size()):
+            token = self.byte_level_bpe.tokenizer.id_to_token(i)
+            self.add_token(token)
+            self.freq[token] = 1
```

Comment:
is there not a smarter value we can use?

Reply:
Could you elaborate on this?

Reply:
resolved offline